.gitignore

```diff
@@ -3,3 +3,5 @@ build/
 .DS_Store
 src/test/config/config.properties
 /target/
+.idea
```

Comment:
These changes to .gitignore should be left out of this commit. Feel free to submit them as a separate change though.